### ISSUE TITLE
VoiceOver: Video messages are not showing in the messages list

### ISSUE THREAD

**Issue by MarcoZehe** [TelegramMessenger/Telegram-iOS#145]
STR:
1. Open a group chat or single chat that has at least one video message.
2. Turn on VoiceOver.
3. Try to find the message containing the video to play it.
   * Actual: Swiping left and right jumps over the video message. If exploring the screen at the position where the video message should be, the screen is blank, at least in VoiceOver's view.
   * Expected: VoiceOver should "see" this like any other message and be able to activate video playback.
